persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -289,3 +289,9 @@ WHERE qualified_id = ?;
 
 getGuestRoomLinkByConversationId:
 SELECT guest_room_link FROM Conversation WHERE qualified_id = ?;
+
+selectConversationIdsWithoutMetadata:
+SELECT qualified_id
+FROM Conversation
+WHERE type IS NOT 'SELF'
+AND (name IS NULL OR creator_id = '');
```

Comment:
So we rely on the `name` being null as indicator for missing metadata? What if we have conversation without names? Have you considered adding a dedicated boolean field on the user and conversation entity which signals that metadata is missing?

Reply:
I know that iOS is currently doing something similar, and I wanted to avoid this, since I saw a nearly impossible way to have an empty named conversation 🤔 

nvm, I can add the field

Reply:
> I know that iOS is currently doing something similar, and I wanted to avoid this, since I saw a nearly impossible way to have an empty named conversation 🤔
> 
> nvm, I can add the field

We never now what happens in future.

Btw 1:1 team conversations are group conversations without a name.